packages/design/src/FormManager/FormEdit/components/PreviewSequencePattern/DraggableList.tsx

```diff
@@ -58,47 +66,109 @@ export const DraggableList: React.FC<DraggableListProps> = ({
             updateOrder(newOrder);
           }
         }}
+        onDragCancel={() => {
+          setActiveId(null);
+        }}
       >
         <SortableContext items={order} strategy={verticalListSortingStrategy}>
           {arrayChildren.map((child, index) => {
             const patternId = order[index];
             return (
-              <SortableItem key={patternId} id={patternId}>
+              <SortableItem
+                key={patternId}
+                id={patternId}
+                isActive={patternId === activeId}
+                isOver={patternId === activeId}
+              >
                 {child}
               </SortableItem>
             );
           })}
         </SortableContext>
+
+        <DragOverlay>
+          {activeId ? (
+            <SortableItemOverlay>
+              {arrayChildren[order.indexOf(activeId)]}
+            </SortableItemOverlay>
+          ) : null}
+        </DragOverlay>
       </DndContext>
     </div>
   );
 };
 
+const SortableItemOverlay = ({
+  children,
+}: {
+  children: React.ReactNode;
+}) => {
+  const context = useFormManagerStore((state) => state.context);
+
+  return (
+    <div
+      className={`${styles.draggableListWrapper} draggable-list-item-wrapper bg-white margin-bottom-3 dragging`}
+      style={{
+        boxShadow: '0 16px 24px rgba(0, 0, 0, 0.4)',
+        cursor: 'grabbing',
+      }}
+    >
+      <div className="grid-row draggable-list-item">
+        <div
+          className="grid-col-12 width-full draggable-list-button padding-2"
+          style={{ outline: '.25rem solid #783cb9' }}
+        >
+          <svg
+            className="usa-icon margin-x-auto display-block"
+            aria-hidden="true"
+            focusable="false"
+            role="img"
+          >
+            <use
+              xlinkHref={`${context.uswdsRoot}img/sprite.svg#drag_handle`}
+            ></use>
+          </svg>
+        </div>
+        <div className="grid-col-12 grid-col">{children}</div>
+      </div>
+    </div>
+  );
+};
+
 const SortableItem = ({
   id,
   children,
+  isActive,
+  isOver,
 }: {
   id: UniqueIdentifier;
   children: React.ReactNode;
+  isActive: boolean;
+  isOver: boolean;
 }) => {
   const { attributes, listeners, setNodeRef, transform, transition } =
     useSortable({ id });
-  const context = useFormManagerStore(state => state.context);
+  const context = useFormManagerStore((state) => state.context);
 
   return (
     <div
-      className={`${styles.draggableListWrapper} draggable-list-item-wrapper bg-white margin-bottom-3`}
+      className={`${styles.draggableListWrapper} draggable-list-item-wrapper bg-white margin-bottom-3 ${
+        isActive ? 'dragging' : ''
```

Comment:
Where is the `dragging` class defined?

Reply:
I originally added that class name along with "over" with the intention of putting the styles in the module css. Unfortunately, the styles weren't being picked up, so I ended up adding them inline. I removed the class names.